### DISCUSSION TITLE
Frequent releases on App Store without proper release notes

### DISCUSSION THREAD

**Discussion by sbaishya** [mshibanami/redirect-web#41] [Questions] [answered]
I'm seeing updates to this extension on a near-daily frequency, without an adequate description of what has changed. The extension has gone from v6.0 to 6.9 in 4 weeks with each new release having exactly the same change description as the original release, whereas before v 6.0 releases were less frequent but the specific changes were highlighted.

Is this frequency of change expected / planned or is something unexpected happening?

Please can you revert to including proper release notes in the App Store so that it is clear what has been changed?

**Comment by mshibanami** [comment, marked as answer]
@sbaishya Thank you for your feedback! And sorry for the confusion.

There is barely anything unexpected happening. I’ve just got enough time to work on this project and have improved small parts of the app daily for the past month. For example, I adjusted the app icon a bit in the latest release.

The daily improvements are mentioned as “there are other improvements” in the release notes. But yeah, I agree it doesn’t say much.
I’ll reconsider the current workflow, but sorry, please don’t expect to see the improvement very soon since I’d need to build a workflow. Probably I’ll address it in the next major update for declarativeNetRequest.

**Reply by sbaishya** [reply]
Thanks @mshibanami, I appreciate you replying
